tools/lumpy/lumpy.xml

```diff
@@ -160,21 +135,21 @@
     </inputs>
 
     <outputs>
-        <data format="tabular" name="histogram" label="Lumpy on ${on_string}: Fragment size distribution">
+        <data format="tabular" name="histogram" label="Lumpy on ${on_string}: Fragment size distribution" from_work_dir="input.lib.histo">
```

Comment:
Again "for my culture": what happens to files that are in the from_work_dir but are not collected through this tag ? No need to clean it with an rm ? (I see you do not use this command anymore)